Classes/Domain/Repository/LogRepository.php

```diff
@@ -85,9 +85,9 @@ public function findByTime($starttime, $endtime)
      *
      * @return array array(<taskUid> => array('task' => <task>, 'logs' => array(<log>, ...) ), ...)
      */
-    public function findGroupedByTask()
+    public function findGroupedByTask(int $startTime, int $endTime)
     {
-        $logs = $this->findAll();
+        $logs = $this->findByTime($startTime, $endTime);
```

Comment:
should we check here `$startTime > $endTime` ?

Reply:
The findByTime() function does that in the query.